torchbenchmark/models/clip/install.py

```diff
@@ -0,0 +1,18 @@
+import os
+import subprocess
+import sys
+
+def pip_install_requirements():
+    subprocess.check_call([sys.executable, '-m', 'pip', 'install', '-q', '-r', 'requirements.txt'])
+
+def download_data():
+    subprocess.check_call(['wget', '-O', './data', 'https://upload.wikimedia.org/wikipedia/commons/thumb/9/91/Pizza-3007395.jpg/2880px-Pizza-3007395.jpg'])
```

Comment:
any legal requriements on the image? i was just using a random one in my notebook

Reply:
it's a cc0 image from the wikipedia page for pizza lol https://en.wikipedia.org/wiki/Pizza#/media/File:Pizza-3007395.jpg so should be good